internal/provisioner/plain/controllers/bundle_controller.go

```diff
@@ -455,6 +456,14 @@ func bundleGitRepoPod(pod *corev1.Pod, source rukpakv1alpha1.GitSource, unpackIm
 	return pod, nil
 }
 
+func (r *BundleReconciler) getCredential(ctx context.Context, secretname string) (string, string, error) {
+	secret, err := r.KubeClient.CoreV1().Secrets(r.PodNamespace).Get(ctx, secretname, metav1.GetOptions{})
```

Comment:
would the namespace be the one the Bundle is in versus the one the BundleReconciler (plain provisioner) is in?

Reply:
It is the namespace where the plain provisioner running.  Bundle is cluster scope, isn't it?

Reply:
yup, sorry blanked on that haha, they are cluster scoped. 

so we would expect a user to create a secret in the `rukpak-system` namespace for example for the provisioner to be able to source a bundle that references some particular private git content. I suppose that UX makes sense.

Reply:
I'm cool with requiring that Secrets must live in the rukpak system namespace in order to use this functionality. I'm mainly interested in avoiding an implementation where we copy Secrets between namespaces.